client-ui/src/app/components/lineage/lineage-details/operation-properties-details/operation-properties-details.component.ts

```diff
@@ -13,18 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { AfterViewInit, ChangeDetectorRef, Component, ComponentFactoryResolver, OnDestroy, QueryList, ViewChildren, ViewContainerRef } from '@angular/core';
-import { Store } from '@ngrx/store';
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  ComponentFactoryResolver,
+  OnDestroy,
+  QueryList,
+  ViewChildren,
+  ViewContainerRef
+} from '@angular/core';
+import {Store} from '@ngrx/store';
 import * as _ from 'lodash';
-import { Observable, Subscription } from 'rxjs';
-import { map, switchMap } from 'rxjs/operators';
-import { AppState } from 'src/app/model/app-state';
-import { Property, PropertyType } from 'src/app/model/property';
-import { OperationType, PropertiesComponents } from 'src/app/model/types/operationType';
-import { AttributeVM } from 'src/app/model/viewModels/attributeVM';
-import { OperationDetailsVM } from 'src/app/model/viewModels/operationDetailsVM';
-import { getOperationIcon, getOperationColor } from 'src/app/util/execution-plan';
-import { getText } from 'src/app/util/expressions';
+import {Observable, Subscription} from 'rxjs';
+import {map, switchMap} from 'rxjs/operators';
+import {AppState} from 'src/app/model/app-state';
+import {Property, PropertyType} from 'src/app/model/property';
+import {OperationType, PropertiesComponents} from 'src/app/model/types/operationType';
+import {AttributeVM} from 'src/app/model/viewModels/attributeVM';
+import {OperationDetailsVM} from 'src/app/model/viewModels/operationDetailsVM';
+import {getOperationColor, getOperationIcon} from 'src/app/util/execution-plan';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600675)